src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,21 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+import './PostInfo.scss';
+
+export const PostInfo = ({ post: { title, user, body, comments } }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{title}</h3>
+
+      <p>
+        {' Posted by  '}
+
+        <UserInfo user={user} />
```

Comment:
find  can returns undefined, let's use add the check here
```suggestion
        {user && <UserInfo user={user} />}
```
<img width="512" alt="image" src="https://github.com/mate-academy/react_static-list-of-posts-js/assets/70220667/9f4192df-1c5f-4c08-bdd6-ae38dfcc941a">